appserver/distributions/web/src/test/java/org/glassfish/distributions/test/EmbeddedTest.java

```diff
@@ -36,12 +37,16 @@
 import org.glassfish.embeddable.web.HttpListener;
 import org.glassfish.embeddable.web.WebContainer;
 import org.glassfish.embeddable.web.WebListener;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 
 public class EmbeddedTest {
```

Comment:
Yet don't forget to rename both test classes to *IT.java - then failsafe will find and execute them automatically.

Reply:
Maybe better to delete the tests. They haven't been executed for like forever, and didn't even compile before.